### PR TITLE
[14.0] [mig] pos_order_remove_line

### PR DESCRIPTION
**Pull request by fshah-initos**

### CODE REVIEW

File: pos_order_remove_line/static/src/js/orderline.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright 2019 LevelPrime
+ *  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+ */
+
+odoo.define("pos_order_remove_line.Orderline", function (require) {
+    "use strict";
+
+    var Orderline = require("point_of_sale.Orderline");
```
Comment (ivantodorovich) [inline pos_order_remove_line/static/src/js/orderline.js]:
```suggestion
    const Orderline = require("point_of_sale.Orderline");
```

File: pos_order_remove_line/static/src/js/orderline.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    const Registries = require("point_of_sale.Registries");
+
+    const PosOrderline = (Orderline) =>
+        class extends Orderline {
+            removeLine() {
+                var line = this.props.line;
+                if (line) {
+                    line.set_quantity("remove");
+                }
```
Comment (ivantodorovich) [inline pos_order_remove_line/static/src/js/orderline.js]: The one liner should be just fine, no?

```suggestion
                this.props.line.set_quantity("remove");
```

**Review by ivantodorovich — APPROVED** [review]
LGTM :+1: 

Just some not blocking comments

**Review by dsolanki-initos — APPROVED** [review]
LGTM

**Pull request merged**